Description: replace ava by tape in test
Author: Xavier Guimard <yadd@debian.org>
Forwarded: not-needed
Last-Update: 2022-10-27

--- a/packages/alias/test/test.mjs
+++ b/packages/alias/test/test.mjs
@@ -2,7 +2,7 @@
 import { fileURLToPath } from 'url';
 import { createRequire } from 'module';
 
-import test from 'ava';
+import test from 'tape';
 import { rollup } from 'rollup';
 import nodeResolvePlugin from '@rollup/plugin-node-resolve';
 
@@ -67,18 +67,21 @@
 
 test('type', (t) => {
   t.is(typeof alias, 'function');
+  t.end();
 });
 
 test('instance', (t) => {
   const result = alias();
   t.is(typeof result, 'object');
   t.is(typeof result.resolveId, 'function');
+  t.end();
 });
 
 test('defaults', (t) => {
   const result = alias({});
   t.is(typeof result, 'object');
   t.is(typeof result.resolveId, 'function');
+  t.end();
 });
 
 test('Simple aliasing (array)', (t) =>
@@ -265,6 +268,7 @@
 
 test('Global customResolver function', (t) => {
   const customResult = 'customResult';
+  t.plan(1);
 
   return resolveAliasWithRollup(
     {
@@ -288,6 +292,7 @@
 test('Local customResolver function', (t) => {
   const customResult = 'customResult';
   const localCustomResult = 'localCustomResult';
+  t.plan(1);
 
   return resolveAliasWithRollup(
     {
@@ -311,6 +316,7 @@
 
 test('Global customResolver plugin-like object', (t) => {
   const customResult = 'customResult';
+  t.plan(1);
 
   return resolveAliasWithRollup(
     {
@@ -334,6 +340,7 @@
 test('Local customResolver plugin-like object', (t) => {
   const customResult = 'customResult';
   const localCustomResult = 'localCustomResult';
+  t.plan(1);
 
   return resolveAliasWithRollup(
     {
@@ -480,6 +487,7 @@
       ]
     ]);
     t.deepEqual(result, ['entry-point', 'non-entry-point']);
+    t.end();
   });
 });
 
@@ -530,6 +538,7 @@
       ]
     ]);
     t.deepEqual(result, ['entry-point', 'non-entry-point']);
+    t.end();
   });
 });
 
@@ -538,6 +547,7 @@
     entry: 0,
     option: 0
   };
+  t.plan(1);
   return resolveAliasWithRollup(
     {
       entries: [
@@ -607,4 +617,5 @@
         `expected ${normalizedIds[index]} to end with ${expectedId}`
       )
     );
+  t.end();
 });
